### PR TITLE
Update GoFSH for 0.1.0 release

### PR DESCRIPTION
**Pull request by cmoesel**
- Update package.json w/ new name, version, description, contributors, bin, and publisher-related tasks
- Convert GitLab CI config to GitHub CI config
- Update README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "fsh-ihsus",
-  "version": "0.0.1",
-  "description": "Ichthymorphic Helper Systematically Undoes Sushi (FSH Decompiler)",
+  "name": "gofsh",
+  "version": "0.1.0",
+  "description": "GoFSH is a FHIR Shorthand (FSH) decompiler, able to convert formal FHIR definitions from JSON/XML to FSH.",
```
Comment (ngfreiter) [inline package.json:4]: I don't think we can yet claim XML to FSH.

**Review by ngfreiter — COMMENTED** [review]
One small change, looks good otherwise!

File: package.json (hunk 1 above)
Comment (cmoesel) [reply package.json:4]: I think I was thinking of the description more as the target than the current reality.  But point taken.

Comment (cmoesel) [reply package.json]: Addressed in 53dc60d

**Review by ngfreiter — APPROVED** [review]
Take a hike XML!

**Pull request merged**